include/oneapi/dpl/pstl/glue_algorithm_ranges_impl.h

```diff
@@ -33,6 +33,28 @@ namespace experimental
 namespace ranges
 {
 
+// [alg.any_of]
+
+template <typename _ExecutionPolicy, typename _Range, typename _Predicate>
+oneapi::dpl::__internal::__enable_if_execution_policy<_ExecutionPolicy, bool>
+any_of(_ExecutionPolicy&& __exec, _Range&& __rng, _Predicate __pred)
+{
+    return oneapi::dpl::__internal::__ranges::__pattern_any_of(::std::forward<_ExecutionPolicy>(__exec),
+                                                               views::all(::std::forward<_Range>(__rng)), __pred);
```

Comment:
`views::all` --> `views::all_read`?
By the way, don't we need to use `oneapi::dpl::...::view` here?

Reply:
1) yes...
2) Currently, it doesn't produce any issues... I suggest return to that question when we get out the functionally form experimental namespace, at least.

Reply:
ok